.github/workflows/components-contrib-all.yml

```diff
@@ -28,9 +28,38 @@ on:
     branches:
       - release-*
 jobs:
+  post-comment:
```

Comment:
Consider adding an `if` directly in the job to avoid the entire job from running if the event is not repository_dispatch? https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idif

Reply:
I just made another update -- I do need this job to run now (you will see why).

Reply:
Oh I think I also might need to use outputs / inputs. One moment